core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -122,20 +126,24 @@ class AdminClient(val time: Time,
   }
 
   def describeGroup(groupId: String): GroupSummary = {
-    val coordinator = findCoordinator(groupId)
-    val responseBody = send(coordinator, ApiKeys.DESCRIBE_GROUPS, new DescribeGroupsRequest(List(groupId).asJava))
-    val response = new DescribeGroupsResponse(responseBody)
-    val metadata = response.groups().get(groupId)
-    if (metadata == null)
-      throw new KafkaException(s"Response from broker contained no metadata for group $groupId")
-
-    Errors.forCode(metadata.errorCode()).maybeThrow()
-    val members = metadata.members().map { member =>
-      val metadata = Utils.readBytes(member.memberMetadata())
-      val assignment = Utils.readBytes(member.memberAssignment())
-      MemberSummary(member.memberId(), member.clientId(), member.clientHost(), metadata, assignment)
-    }.toList
-    GroupSummary(metadata.state(), metadata.protocolType(), metadata.protocol(), members)
+    findCoordinator(groupId) match {
+      case None => 
+        throw new KafkaException(s"Offfsets topic is unavailable as it takes little time to be created on a new cluster")
```

Comment:
Changed the message. Also did a println instead of exception since it was printing the stacktrace; and in turn changed the output of the function to an option. Is it better now?